debian/rules

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/make -f
 
-
+INIT_VARIANT	?= systemd
 VERSION  := $(shell dpkg-parsechangelog|grep ^Version|awk '{print $$2}')
 UVERSION := $(shell echo $(VERSION)|sed 's/-[[:digit:]]\+$$//')
 
-ifneq ($(wildcard /usr/bin/dh_systemd_start),)
+ifeq ($(INIT_VARIANT),systemd)
```

Comment:
Maybe now is good time to just enable it unconditionally. All Debian and Ubuntu distros, which offers something else by default, already gone.